pyxel/core/src/pyxelcore/input.cc

```diff
@@ -18,134 +19,137 @@ Input::Input() {
 
   is_mouse_visible_ = false;
 
-  for (int32_t i = 0; i < KEY_COUNT; i++) {
+  for (int32_t i = 0; i < KEY_COUNT; i++) 
+  {
     key_state_[i] = 0;
   }
 }
 
-Input::~Input() {
+Input::~Input() 
+{
   SDL_FreeCursor(blank_cursor_);
 
-  if (gamepad1_) {
+  if (gamepad1_) 
+  {
     SDL_GameControllerClose(gamepad1_);
   }
 
-  if (gamepad2_) {
+  if (gamepad2_) 
+  {
     SDL_GameControllerClose(gamepad2_);
   }
 }
 
-void Input::Update(Window* window, int32_t frame_count) {
+void Input::Update(Window* window, int32_t frame_count) 
+{
   frame_count_ = frame_count + 1;  // change frame_count to start from 1
 
   SDL_GetGlobalMouseState(&mouse_x_, &mouse_y_);
 
-  mouse_x_ = (mouse_x_ - (window->WindowX() + window->ScreenX())) /
-             window->ScreenScale();
-  mouse_y_ = (mouse_y_ - (window->WindowY() + window->ScreenY())) /
-             window->ScreenScale();
+  mouse_x_ = (mouse_x_ - (window->WindowX() + window->ScreenX())) / window->ScreenScale();
```

Comment:
I think the formula was deliberately setup as two lines so make it clear what is on the top and what is on the bottom. Now this is less clear.